src/replicate_stage.rs

```diff
@@ -78,10 +80,17 @@ impl ReplicateStage {
         let mut res = Ok(());
         let last_entry_id = {
             let mut num_entries_to_write = entries.len();
+            let mut last_entry_id_ = *last_entry_id;
```

Comment:
you can re-use the name if you like?

Reply:
I wanted some visible separation even if rust was ok with it.

Reply:
ick ;)